main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_cloudtrail" "trail" {
+  name           = "trail${replace(title(var.project), " ", "")}${title(var.environment)}"
+  s3_bucket_name = "${var.s3_bucket_name}"
+
+  enable_logging                = true
+  include_global_service_events = true
+  is_multi_region_trail         = false
+  enable_log_file_validation    = false
+
+  tags {
+    Name        = "${var.project}"
+    Environment = "${var.environment}"
+  }
+
+  depends_on = ["aws_s3_bucket.trail"]
+}
+
+resource "aws_s3_bucket" "trail" {
+  count = "${var.create_s3_bucket ? 1 : 0}"
+
+  bucket = "${var.s3_bucket_name}"
+  region = "${var.region}"
+
+  force_destroy = true
```

Comment:
Setting this to `true` makes me feel uneasy. I think that we should leave it unassigned so that it inherits the default of `false`.